orm/authjs-nextjs/README.md

```diff
@@ -0,0 +1,116 @@
+# Auth.js + Prisma example
+
+This example shows how to implement **authentication** using [Auth.js](https://authjs.dev/), [Next.js](https://nextjs.org/) and [Prisma](https://www.prisma.io).
+
+## Getting started
+
+### 1. Download example and navigate into the project directory
+
+Download this example:
+
+```
+npx try-prisma@latest --template orm/authjs-nextjs
+```
+
+Then, navigate into the project directory:
+
+```
+cd authjs-nextjs
+```
+
+<details><summary><strong>Alternative:</strong> Clone the entire repo</summary>
+
+Clone this repository:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+```
+
+Install npm dependencies:
+
+```
+cd prisma-examples/orm/authjs-nextjs
+npm install
+```
+
+</details>
+
+Remove `.example` from both `env.example` and `.env.local.example`
+
+### 2. Create a Prisma Postgres instance
+
+This example uses a [Prisma Postgres](https://prisma.io/postgres) database by default. To get started with the project, you will need to setup a Prisma Postgres connection string:
+
+#### Option 1. Use the Prisma Data Platform Console
+
+1. Set up a new Prisma Postgres instance in the [Prisma Data Platform Console](https://console.prisma.io) and copy the database connection URL.
+
+2. Add your database url to the `.env`
+
+#### Option 2. Use `npx create-db`
+
+1. Run `npx create-db@latest` in your terminal.
+
+2. Copy the string labeled **"Use this connection string optimized for Prisma ORM"** and add it to the `.env` under `DATABASE_URL`
+
+That's it, your project is now configured to use Prisma Postgres!
+
+### 3. Generate and migrate Prisma client
+
+1. Run the following command to generate the Prisma Client. This is what you will be using to interact with your database.
+
+```
+npx prisma generate
+```
+
+2. Migrate the DB
+
+```
+npx prisma migrate dev --name init
+```
+
+### 4. Set up Auth.js
+
+1. Generate a Auth.js secret
+
+```
+npx auth secret
+```
+
+2. Add the secret to the `.env`.
+
+### 5. Set up Github OAuth
+
+There are many options for OAuth providers (See list here). For this example we've gone with Github.
+
+1. Navigate to [Github's Developer Settings](https://github.com/settings/developers) and hit **New OAuth app**
+
+2. Fill out the required details and hit **Register application**
+
+- Applciation name: "Auth.js + Prisma Example"
+- Homepage URL: "http://localhost:3000/"
+- Authorization callback URL: "http://localhost:3000/api/auth/callback/github"
+
+4. Hit **Generate a new client secret** to create a new Client Secret
+
+5. Paste the Client ID and Client Secret in `.env.local` next to their respective variables
+
+### 6. Start the development server
+
+```
+npm run dev
+```
+
+The server is now running at http://localhost:3000
```

Comment:
I followed the instructions but ran into this error:

<img width="1512" height="837" alt="Screenshot 2025-08-07 at 2 40 23 PM" src="https://github.com/user-attachments/assets/6d6f3f57-8b23-47b9-96aa-d9fe7b7bab06" />

Reply:
I still get the same error. The error goes away of I remove `--turbopack` from the `dev` command.